client/ayon_core/modules/deadline/plugins/publish/collect_user_credentials.py

```diff
@@ -3,32 +3,85 @@
 
 Requires:
     context -> project_settings
+    instance.data["deadline"]["url"]
+        or instance.data["deadlineUrl"] for backward compatibility, remove soon
 
 Provides:
-    context -> deadline_require_authentication (bool)
-    context -> deadline_auth (tuple (str, str)) - (username, password) or None
+    instance.data["deadline"] -> require_authentication (bool)
+    instance.data["deadline"] -> auth (tuple (str, str)) -
+        (username, password) or None
 """
 import pyblish.api
 
+from ayon_api import get_server_api_connection
+from ayon_core.modules.deadline.deadline_module import DeadlineModule
+from ayon_core.modules.deadline import __version__
 
-class CollectDeadlineUserCredentials(pyblish.api.ContextPlugin):
+
+class CollectDeadlineUserCredentials(pyblish.api.InstancePlugin):
     """Collects user name and password for artist if DL requires authentication
     """
 
     # Run before collect_deadline_server_instance.
```

Comment:
Is this comment still relevant with the order change?

Reply:
Removed.